content/en/docs/reference/glossary/cgroups.md

```diff
@@ -1,13 +1,13 @@
 ---
-title: CGroup
+title: cgroup
 id: cgroup
 date: 2019-06-25
 full_link:
 short_description: >
-  cgroup (abbreviated from control group) is a Linux kernel feature that limits, accounts for, and isolates the resource usage (CPU, memory, disk I/O, network, etc.) for a collection of processes.
+  cgroup (abbreviated from control group) is a Linux kernel feature that limits, accounts for, and isolates the resource usage (CPU, memory, disk I/O, network) for a collection of processes.
 
 aka:
 tags:
 - fundamental
 ---
-cgroup (abbreviated from control group) is a Linux kernel feature that limits, accounts for, and isolates the resource usage (CPU, memory, disk I/O, network, etc.) for a collection of processes.
+cgroup (abbreviated from control group) is a Linux kernel feature that limits, accounts for, and isolates the resource usage (CPU, memory, disk I/O, network) for a collection of processes.
```

Comment:
What do you think about removing the phrase `(abbreviated from control group)` and instead adding `(control group)` to the glossary title?

Reply:
I can see cgroup mentioned in docs and as a result, the search in glossary would be for 'cgroup'. This is the reason I would keep the title as cgroup.

Reply:
OK, I think you could still use cgroup in the doc text,
`{{< glossary_tooltip text="cgroup" term_id="cgroup" >}}`.

I realize that this is a minor point, but do you think it makes sense to rename the file to cgroup.md instead of cgroups.md?